plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -79,6 +79,16 @@ const (
 // For example:
 // https://github.com/kubernetes/kubernetes/blob/36a218e/plugin/pkg/scheduler/factory/factory.go#L422
 
+var (
+	predicatesOrdering = []string{"CheckNodeCondition",
```

Comment:
We should also add a comment that this list must be updated when new predicates are added.

Reply:
indeed